coretex/cli/modules/node.py

```diff
@@ -307,7 +371,10 @@ def initializeNodeConfiguration() -> None:
     config = loadConfig()
 
     if isNodeConfigured(config):
-        return
+        if isConfigurationValid(config):
```

Comment:
Aren't these two (`isNodeConfigured` and `isConfigurationValid`) the same function?

Reply:
Well right now, these 2 functions are different. isNodeConfigured() function is only checking for required fields for node in config and isConfigurationValid function is checking if each field has valid type and value (if must be checked). Do you want this to be changed and in what way?

Reply:
We can combine the 2 functions into a single one

Reply:
After short discussion we decided to keep two functions for now.